src/services/router.ts

```diff
@@ -21,7 +21,7 @@ export const addLiquidityByID = (id: string) => `${liquidityBase}${id}`;
 export const portfolioRewardsStakeByID = (id: string) =>
   `${rewardsBase}stake/${id}`;
 export const portfolioRewardsStakeByIDnPos = (id: string, pos: string) =>
-  `${rewardsBase}${id}?posGroupId=${pos}`;
+  `${rewardsBase}stake/${id}?posGroupId=${pos}`;
```

Comment:
note that `${rewardsBase}stake` is used in 3 places and maybe we should just do a `stakeRewardsBase`